src/plugins/discover/public/application/main/components/sidebar/discover_field.tsx

```diff
@@ -396,33 +452,57 @@ function DiscoverFieldComponent({
   }
 
   const renderPopover = () => {
-    const details = getDetails(field);
-
-    // TODO: integrate <FieldStats .../>
+    const dateRange = data?.query?.timefilter.timefilter.getAbsoluteTime();
```

Comment:
It's nothing nothing to be done now, but this seems like a good candidate to extract to a separate component (given that discover_field.tsx grew and also other candidates for refactoring here )

Reply:
@kertal Can look into it as a part of https://github.com/elastic/kibana/issues/140363

Reply:
makes sense! thx!